yarn-project/cli/src/cmds/example_contracts.ts

```diff
@@ -4,6 +4,6 @@ import { getExampleContractArtifacts } from '../utils.js';
 
 export async function exampleContracts(log: LogFn) {
   const abisList = await getExampleContractArtifacts();
-  const names = Object.keys(abisList);
+  const names = Object.keys(abisList).filter(name => name !== 'AvmTestContractArtifact');
   names.forEach(name => log(name));
```

Comment:
Filtering out the avm test contract so it doesn't show in `aztec-cli example-contracts`

Reply:
It could be worth generalising this to `.contains("Avm")`